### DISCUSSION TITLE
English Monarchs and Marriages

### DISCUSSION THREAD

**Discussion by nrennie** [qmd-lab/closeread#90] [Show and tell]
I had so much fun experimenting with this @andrewpbray! Just wanted to share a quick scrollytelling visualisation of some TidyTuesday data on English monarchs. Happy to help with testing/docs if that's helpful!

![monarchs](https://github.com/user-attachments/assets/5e08440a-5ab9-4c61-82bb-391fd6ded8bb)

Online version: https://nrennie.rbind.io/scrollytelling/
Source: https://github.com/nrennie/scrollytelling

**Comment by andrewpbray** [comment]
This is lovely! It's a great use-case too: that's a very complex graphic that warrants a close read.

We'll ping you during our next development phase in the next 2-3 weeks to see if you have a moment to help test some new features - thanks!

**Comment by kpivert** [comment]
[Nicola](@nrennie), 

This is fantastic and very effective. Thank you for sharing your code. 

I had one question, if you have time. Was there a specific reason why you chose to use the 'pre-rendered' PNGs of your visualizations in the closeread instead of individual code cells (with your `period_plot()` function)? 

Totally understand if not. Thanks again (and thanks to @andrewpbray and @jimjam-slam for the great extension).

Kurtis

**Reply by nrennie** [reply]
Hi @kpivert, thanks so much! I used pre-rendered PNG files because I had some layout issues when using plots that were outputs directly from code blocks within the quarto doc - essentially issue #78.

**Reply by kpivert** [reply]
Got it! Thanks [Nicola](@nrennie) for your quick reply and clarification. Now I understand what I was experiencing. 

Thanks again!
